src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -146,6 +146,30 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             }
         }
 
+        if (const auto& child{ GetTemplateChild(L"Expander") })
+        {
+            if (const auto& expander{ child.try_as<Microsoft::UI::Xaml::Controls::Expander>() })

```

Comment:
This is just to ensure we have the correct control, right? If there was a template child named Expander we wouldn't want to give it these properties _unless it actually was an expander_?

Reply:
Yeah I like to think of this as just a sanity check! Would rather have it than not though